src/components/Card/Card.js

```diff
@@ -17,6 +18,9 @@ export default function Card(image, name, house) {
   <h2>${name}</h2>
   <span class="Card_housename">${house}</span>
  `
+  if (alive === false) {
```

Comment:
You could also write it like this:
```suggestion
  if (!alive) {
```